crates/core/tedge/src/cli/connect/c8y_direct_connection.rs

```diff
@@ -15,6 +15,7 @@ pub fn create_device_with_direct_connection(
 
     let mut mqtt_options = MqttOptions::new(bridge_config.remote_clientid.clone(), host[0], 8883);
     mqtt_options.set_keep_alive(std::time::Duration::from_secs(5));
+    mqtt_options.set_connection_timeout(60);
```

Comment:
Why not using `CONNECTION_TIMEOUT` here instead of the hard-code `60`?

Reply:
Though what can't we make the value configurable? Having the configuration in a config file means that we can change it without code changes in the future.

Reply:
Sure, that's a good idea. @didier-wenzek what do you say?

Reply:
> Why not using `CONNECTION_TIMEOUT` here instead of the hard-code `60`?

Yes, make sense. Updated the code.